flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/source/config/PostgresSourceOptions.java

```diff
@@ -21,46 +21,12 @@
 
 import com.ververica.cdc.connectors.base.options.JdbcSourceOptions;
 import com.ververica.cdc.connectors.postgres.source.PostgresSourceBuilder;
-import com.ververica.cdc.debezium.table.DebeziumChangelogMode;
 
 import java.time.Duration;
 
 /** Configurations for {@link PostgresSourceBuilder.PostgresIncrementalSource}. */
 public class PostgresSourceOptions extends JdbcSourceOptions {
 
```

Comment:
Why we move these options to Factory ? I think keeping them in PostgresSourceOptions is more clear.
We can use PG_PORT to avoid member conflicts with parent if you worries the  conflicts